tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -82,79 +101,62 @@ spec:
           repository=$(jq -r '.repository' <<< $component)
           name=$(jq -r '.name' <<< $component)
           git_sha=$(jq -r '.source.git.revision' <<< $component) # this sets the value to "null" if it doesn't exist
-          #
-          # The tag is determined as follows:
-          #
-          # If `floatingTag` is non-empty, we push to $floatingTag and $floatingTag-$timestamp.
+
+          # If `floatingTags` is non-empty, we push each image to $floatingTag and $floatingTag-$timestamp.
```

Comment:
I think this needs a bit of improvement to say that this is for each of the floatingTags, like:

```sh
# If `floatingTags` is non-empty, for each of the `floatingTags` we push each image to $floatingTag and $floatingTag-$timestamp.
```

Maybe?

Reply:
updated